release-notes.md

```diff
@@ -116,43 +122,49 @@ React Native wrappers for the following extensions were pushed to npmjs:
 
 ## May 6, 2019
 
-The following updates were made to the Mobile Core extension in Android version 1.3.0:
+**Android Core 1.3.0**
 
 * The `MobileCore.start()` API should only get called once, and the SDK ignores the subsequent calls to this API.
 * Added an internal API for React Native support.
 
 ## May 2, 2019
 
-The following update was made to the Mobile Core extension in iOS version 2.2.1:
+The following updates were made in this release:
+
+**iOS Core 2.2.1**
 
 * Fixed an issue when the app network crashes under extreme conditions.
 
+**Android Places 1.1.0**
+
+- Introduced a new API for getNearByPlaces with errorCallback, called with errorCode indicating the reason for error
```

Comment:
Please reword as follows:

"Introduced a new API for `getNearByPlaces`, which has an errorCallback and is called with an `errorCode` that indicates the reason for the error."